test/functionwrappers.jl

```diff
@@ -12,7 +12,7 @@ let
         rateinterval = (u, p, t) -> 0.1)
 
     prob = DiscreteProblem([0.0], (0.0, 2.0), [1.0])
-    jprob = JumpProblem(prob, Coevolve(), jump; dep_graph = [[1]], rng)
+    jprob = JumpProblem(prob, Coevolve(), jump; vr_aggregator = VR_FRM(), dep_graph = [[1]], rng)
```

Comment:
Why? This is a purely discrete problem as it uses a bounded variable rate jump, so no variable aggregator should be needed or specified.